### PR TITLE
Add messageService and route to create and get messages

### PR DESCRIPTION
**Pull request by ealexisaraujo**

### CODE REVIEW

File: components/messages/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Response code 404:
+
+```js
+{
+    "statusCode": 404,
+    "error": "Not Found",
+    "message": "Not Found",
+}
+```
```
Comment (hyfi06) [inline components/messages/README.md:41]: i don't find not found error. this is not a possible response.

File: components/messages/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    "updatedAt": "",
+    "read": false,
+    "patientId":""
+    "messageText": ""
+    ]
+  },
+  "message": "Messages retrieved"
+}
+```
+
```
Comment (hyfi06) [inline components/messages/README.md:63]: the response 404 should be here

File: components/messages/README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+console.log(messages);
+/*
+[
+  {
+    _id: 5ed1ddabc0f381982068a797,
+    createdAt: 2020-05-30T04:14:35.043Z,
+    updatedAt: 2020-05-30T04:14:35.043Z,
+    read: false,
+    patientId: 5ecfda99da80860ec24ea439,
+    messageText: 'Listo tu resultado 4'
```
Comment (hyfi06) [inline components/messages/README.md:102]: english message

File: components/messages/README.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+The param `message` should have `patientId`, `messageText` attributes.
+
+Return a ObjectId.
+
+```js
+const messagesService = new MessagesService();
+
+const createUserId = await messagesService.createMessages({
+  patientId: '5ecfda99da80860ec24ea439',
+  messageText: 'Listo tu resultado 4',
```
Comment (hyfi06) [inline components/messages/README.md:119]: english message

File: components/messages/routes.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const passport = require('passport');
+const MessagesService = require('./messagesService');
+
+// strategy JWT
+require('../../utils/auth/strategies/jwt');
+
+function usersApi(app) {
```
Comment (hyfi06) [inline components/messages/routes.js:8]: `messageApi`

File: components/messages/routes.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+  router.get(
+    '/',
+    passport.authenticate('jwt', { session: false }),
+    async function (req, res, next) {
+      const { patientId } = req.query;
+
+      try {
+        const messages = await messagesService.getMessages(patientId);
+        console.log(messages);
```
Comment (hyfi06) [inline components/messages/routes.js:21]: clean this

File: components/messages/README.md (hunk 1 above)
Comment (ealexisaraujo) [reply components/messages/README.md]: fixed

File: components/messages/README.md (hunk 2 above)
Comment (ealexisaraujo) [reply components/messages/README.md:53]: fixed

File: components/messages/README.md (hunk 4 above)
Comment (ealexisaraujo) [reply components/messages/README.md:119]: done!

File: components/messages/routes.js (hunk 5 above)
Comment (ealexisaraujo) [reply components/messages/routes.js]: done!

File: components/messages/routes.js (hunk 6 above)
Comment (ealexisaraujo) [reply components/messages/routes.js:21]: done!

File: components/messages/README.md (hunk 4 above)
Comment (hyfi06) [reply components/messages/README.md:119]: `Results ready to download`

File: components/messages/routes.js (hunk 6 above)
Comment (hyfi06) [reply components/messages/routes.js:21]: :eyes:

**Pull request merged**